kolibri/deployment/default/settings/base.py

```diff
@@ -338,6 +344,8 @@
             "level": "INFO",
             "propagate": False,
         },
+        "cherrypy.access": {"handlers": ["file"], "level": "INFO", "propagate": False},
```

Comment:
Is there a reason not to log this to console in parallel to morango and iceqube?

Reply:
no, I was unsure about that so I only put them to "file". I will log the cherrypy messages to console.